### PR TITLE
Merge EDS into amplify-components

### PR DESCRIPTION
**Pull request by mariush2**
Userstory:
- [AB#470408](https://dev.azure.com/Equinor/4319082b-29ba-4111-a20b-c04f55c7e7c6/_workitems/edit/470408)

Tasks:
- [AB#473558](https://dev.azure.com/Equinor/4319082b-29ba-4111-a20b-c04f55c7e7c6/_workitems/edit/473558)
- [AB#473999](https://dev.azure.com/Equinor/4319082b-29ba-4111-a20b-c04f55c7e7c6/_workitems/edit/473999)
- [AB#474000](https://dev.azure.com/Equinor/4319082b-29ba-4111-a20b-c04f55c7e7c6/_workitems/edit/474000)

### CODE REVIEW

File: .storybook/preview.jsx [hunk 1]
```diff
@@ -40,12 +39,13 @@ export const decorators = [
...
-      date: /Date$/,
-    },
-  },
+  viewMode: 'docs',
+  // controls: {
+  //   matchers: {
+  //     color: /(background|color)$/i,
+  //     date: /Date$/,
+  //   },
+  // },
```
Comment (aslakihle) [inline .storybook/preview.jsx:48]: Comment 
```suggestion
```

File: src/components/Inputs/Search/Search.docs.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+# Search
+
+Allows users to locate or refine content based on simple words or phrases.
+
+<Links
+  ariaUrl="https://www.w3.org/TR/2017/REC-wai-aria-1.1-20171214/#searchbox"
+  figmaUrl="https://www.figma.com/file/0bGXR2sCwMVSDNyyzu5BXrO5/UI%E2%80%94User-Interface?node-id=18442%3A38194"
+  sourceUrl="https://github.com/equinor/design-system/blob/develop/packages/eds-core-react/src/components/Search/Search.tsx"
+  edsSBUrl="https://storybook.eds.equinor.com/?path=/docs/inputs-search--docs"
+></Links>
```
Comment (aslakihle) [inline src/components/Inputs/Search/Search.docs.mdx:14]: unnecessary closing tag. Seems like maybe just on this one xD
```suggestion
/>
```

File: src/components/Inputs/Search/Search.docs.mdx [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import { Links } from 'src/storybook';
+import { Primary, Canvas } from '@storybook/blocks';
+import * as ComponentStories from './Search.stories';
+
+# Search
+
```
Comment (aslakihle) [inline src/components/Inputs/Search/Search.docs.mdx:6]: Should this have the "Component from EDS" text?

File: src/components/Typography/Typography.docs.mdx [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+**Component from EDS**
+
+Presents hierarchy and organises information as clearly and efficiently as possible,
+therefore text sizes and styles were developed to balance content density and reading comfort.
+
+<Links
+  figmaUrl="https://www.figma.com/file/0TbIXrrObWj80Cf7KucKYFL0/Design-Tokens?node-id=2%3A4"
+  sourceUrl="https://github.com/equinor/design-system/blob/develop/packages/eds-core-react/src/components/Typography/Typography.tsx"
+  edsSBUrl="https://storybook.eds.equinor.com/?path=/docs/typography-typography--docs"
+></Links>
```
Comment (aslakihle) [inline src/components/Typography/Typography.docs.mdx:16]: Another one 👀 
```suggestion
/>
```

**Review by aslakihle — CHANGES_REQUESTED** [review]

File: src/components/Inputs/Search/Search.docs.mdx (hunk 3 above)
Comment (mariush2) [reply src/components/Inputs/Search/Search.docs.mdx:6]: Yep, adding

**Review by aslakihle — APPROVED** [review]

**Pull request merged**